illstack/CompHaloProperties.py

```diff
@@ -20,15 +20,21 @@ def __init__(self,lims,bins,Linear=False):
         
         self.radbins = np.append(self.r1,self.r2[-1])
 
-    def ComputeHaloProfile(self,pos,quant,weight,volweight=False,stddev=False,innerbin=False):
+    def ComputeHaloProfile(self,pos,quant,weight,scale,volweight=False,stddev=False,innerbin=False,scaled_radius=False):
```

Comment:
could remove scale and just have scaled_radius like on line 60

Reply:
I actually hadn't updated the cumulative version because I don't use it anywhere. It needs to have the input scale along with the option scaled_radius so it knows how much to scale by if scaled_radius is true. So line 60 and the rest of the ComputeCumulativeProfile function need more lines similar to the other function

Reply:
Good good, okay so my comment should be reversed :) add "scale" to ComputeCumulativeProfile etc

Reply:
made the changes to this function, it should be added to the pull request now